tests/functional/cypress.js

```diff
@@ -0,0 +1,12 @@
+const cypress = require('cypress');
+
+cypress.run({
+  headed: process.env.EQ_RUN_FUNCTIONAL_TESTS_HEADLESS ? false : true,
+  config: {
```

Comment:
Instead of duplicating, can we read in?
https://github.com/ONSdigital/eq-survey-runner/blob/4b57796c5051d3dabcb18d8838666813e878a632/tests/functional/cypress/plugins/index.js#L14

Reply:
That plugin file isn't actually used, I've removed it.